fili-core/src/test/java/com/yahoo/bard/webservice/application/JerseyTestBinder.java

```diff
@@ -54,6 +52,7 @@
 import java.util.Map;
 import java.util.Set;
 
+import javax.ws.rs.client.Invocation.Builder;
```

Comment:
Why do we make it a non-static import?

Reply:
That apparently was a mistake.  `Builder` is a class, not a static field so this is correct.

Reply:
I assume static class and static field are all static members. But this is not a big deal anyway